README.md

```diff
@@ -59,6 +61,31 @@ Install specific files in Dom0:
 sudo qubesctl state.apply dotfiles.copy-dom0,dotfiles.copy-sh,dotfiles.copy-vim,dotfiles.copy-x11
 ```
 
+#### Pillar
+
+With Salt, the execution of each state can be opt out via the dotfiles pillar.
+You will need a top and a sls file in your pillar_roots (default to /srv/pillar,
+/srv/pillar/base, can be configured in /etc/salt/minion.d).
+
+Exemple: /srv/pillar/qusal/dotfiles.top to apply the dotfiles sls to all targets:
+```yaml
+base:
+  '*':
+    - qusal.dotfiles
+```
+Exemple: /srv/pillar/dotfiles.sls containing the data to pass to the targets:
+```yaml
+qusal:
+  dotfiles:
+    all: true # This is the default. use 'false' to disable all states.
```

Comment:
Don't do inline comments, just do the pillar file and after it, explain what it is actually doing.

Somewhere it needs to be mentioned that `qusal:dotfiles:all` defaults to true.